site/app/templates/forum/ShowCategories.twig

```diff
@@ -9,127 +9,137 @@
 
-        <ul id="ui-category-list" style="padding-left: 0em;">
-            {# TODO: scrollbar #}
-            {% for category in categories %}
-                <li id="categorylistitem-{{ category.category_id }}" 
-                    class="category-sortable category-item-box" 
-                    style="color: {{ category.color }};"
-                    data-category_desc="{{ category.category_desc }}"
-                    data-category_id="{{ category.category_id }}"
-                >
-                    <i class="fas fa-bars handle" aria-label="Drag to reorder" title="Drag to reorder"></i>
-                    <span class="categorylistitem-desc">
-                            <span class="categorylistitem-desc-text" style="word-wrap: break-word;">{{ category.category_desc }}</span>
-                            <a class="post_button" title="Edit Category Description" aria-label="Edit Category Description"><i class="fas fa-edit category-button edit-category-desc-button"></i></a>
-                        </span>
-                    <span class="categorylistitem-editdesc" style="display: none;">
-                            <input type="text" aria-label="New Description of Category" placeholder="New Description of Category" class="edit-category-desc-input" style="padding: 0;" maxlength="50" onkeyup="checkInputMaxLength($(this))">
-                            <a class="post_button" title="Save Changes" aria-label="Save Changes"><i class="fas fa-check category-button save-desc-button"></i></a>
-                            <a class="post_button" title="Cancel Changes" aria-label="Cancel Changes"><i class="fas fa-times category-button cancel-desc-button"></i></a>
-                        </span>
-                    <div style="float: right;width: auto;">
-                        <select aria-label="Category Color Picker" class='category-color-picker' style="color: white;font-size: 14px;height: 18px;padding: 0px;">
-                            {% for color_name, color_code in category_colors %}
-                                <option value="{{ color_code }}" style="color: white;background-color: {{ color_code }};" {% if color_code == category.color %}selected="selected"{% endif %}>{{ color_name }}</option>
-                            {% endfor %}
-                        </select>
-                        &nbsp;
-                        <a class="post_button" title="Delete Category" aria-label="Delete Category"><i class="fas fa-trash category-button delete-category-button"></i></a>
-                    </div>
-                </li>
-            {% endfor %}
-        </ul>
+				<input class="date_picker" placeholder="Visible Date" name="category_date" id="category_visible_date" type="date">
 
-        </div>
-    </div>
+				<button type="button" title="Add new category" onclick="addNewC{…}ategory('{{ csrf_token }}');" style="margin-left:10px;" class="btn btn-primary btn-sm key_to_click" tabindex="0">
```

Comment:
You have a `{...}` here that is causing issues making it not work and hard to test.